cmd/commands/cli/command.go

```diff
@@ -26,14 +28,28 @@ import (
 
 	"github.com/chzyer/readline"
 	"github.com/mysteriumnetwork/node/cmd"
+	"github.com/mysteriumnetwork/node/core/service"
 	"github.com/mysteriumnetwork/node/metadata"
+	"github.com/mysteriumnetwork/node/services/noop"
+	"github.com/mysteriumnetwork/node/services/openvpn"
+	openvpn_service "github.com/mysteriumnetwork/node/services/openvpn/service"
+	"github.com/mysteriumnetwork/node/services/wireguard"
+	wireguard_service "github.com/mysteriumnetwork/node/services/wireguard/service"
 	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/mysteriumnetwork/node/utils"
 	"github.com/urfave/cli"
 )
 
 const cliCommandName = "cli"
 
+const serviceHelp = `service <action> [args]
+	start	<ProviderID> <ServiceType> [options]
+	stop	<ServiceID>
+	list
+	status	<ServiceID>
+
+	example: service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn --openvpn.port=1194 --openvpn.proto=UDP`
```

Comment:
These instructions looks false

Reply:
I didn't get it. Why does it look false?

Reply:
Probably I got mistaken here